package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed Jan 31 09:25:07 UTC 2018 - ancor@suse.com
+
+- Partitioner: initial support for NFS
+- Partitioner: removed useless tmpfs option
```

Comment:
Please, add some bugzilla/fate nr (I am not that interested, but our bots are :) ).